### DISCUSSION TITLE
How to create a page only for edit certain record only ?

### DISCUSSION THREAD

**Discussion by altimusin** [filamentphp/filament#5358] [Help]
I want to make a new page for customer to editing there own settings. without showing the customer's table

I created the new page with
- php artisan make:filament-page CustomerSetting

![image](https://user-images.githubusercontent.com/43429317/209918325-8606c36a-c2e2-4cde-a383-3e9833336e98.png)

my question are :
1. How to load the data, let say the record id is 10.
2. How to display & edit the data.
3. How to validate
4. How to save the data

I mean, How I can do with correct and easy way?, Should I create CustomerResource first ?
Thank you

**Comment by danharrin** [comment]
Resources are for CRUD only. You just need to use the Form Builder here to render a form. Read the Form Builder docs.

**Reply by altimusin** [reply]
ok
